micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/CamundaBpmVersion.java

```diff
@@ -0,0 +1,21 @@
+package info.novatec.micronaut.camunda.bpm.feature;
+
+import org.camunda.bpm.engine.ProcessEngine;
+
+import java.util.Optional;
+
+public class CamundaBpmVersion {
+    private final String version;
+
+    public CamundaBpmVersion(){
+        this(ProcessEngine.class.getPackage());
+    }
+
+    CamundaBpmVersion(final Package pkg) {
+        this.version = Optional.ofNullable(pkg.getImplementationVersion()).map(String::trim).orElse("");
```

Comment:
* please format this in three lines - like in the original.
* please remove "this" - it is not needed